ptf/lib/grpc_test.py

```diff
@@ -52,11 +57,22 @@ def setUp(self):
         bess_server_addr = testutils.test_param_get("bess_upf_addr")
         self.channel = grpc.insecure_channel(target=bess_server_addr)
         self.bess_client = pb.BESSControlStub(self.channel)
-    
+
     """
     API for reading metrics from BESS-UPF modules
     """
 
+    def sendModuleCommand(self, request, timeout=5, raise_error=True):
```

Comment:
Before, we were not checking the response from BESS, silently dropping errors such as failed QER insertions.